src/Umbraco.Core/Routing/DefaultUrlAssembler.cs

```diff
@@ -0,0 +1,49 @@
+using Umbraco.Cms.Core.Models.PublishedContent;
+
+namespace Umbraco.Cms.Core.Routing;
+
+public class DefaultUrlAssembler : IUrlAssembler
+{
+    private readonly UriUtility _uriUtility;
+
+    public DefaultUrlAssembler(UriUtility uriUtility)
+    {
+        _uriUtility = uriUtility;
+    }
```

Comment:
Just some formatting
```suggestion
public DefaultUrlAssembler(UriUtility uriUtility) => _uriUtility = uriUtility;
```